jobs-shell/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,20 @@
+steps:
+- id: 'Build image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  dir: 'jobs-shell'
+  args: ['build', '-t', 'test', '.'] 
+- id: 'Run image'
```

Comment:
To stay consistent with other samples, please add E2E testing steps here not in a separate script